src/utils/filter-properties.js

```diff
@@ -0,0 +1,12 @@
+const filterProperties = (object, ...properties) => {
+	const keys = Object.keys(object).filter(
+		key => properties.indexOf(key) === -1
```

Comment:
I think* that you can use includes

Reply:
Unfortunately not :( 
I'm using this https://babeljs.io/docs/en/babel-plugin-transform-runtime/ to be able to use async-await. (Read first paragraph)